src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -436,10 +437,12 @@ public Command handleCubeIntakeButton(){
     }
   }
 
-  public Command highScoreSequence() {
+  public Command highScoreSequence(IntakeSubsystem intakeSubsystem) {
     return new SequentialCommandGroup(
+      new InstantCommand(intakeSubsystem::pull),
       goToState(ArmState.HIGHTRANSITION),
       goToState(ArmState.HIGHSCORE),
+      new InstantCommand(intakeSubsystem::stop),
       goToState(ArmState.HIGHDROP)
```

Comment:
ParallelDeadlineCommandGroup is better suited for this. 
Since this factory has multiple subsystems, moving it to Auton or RobotContainer might make more sense

Reply:
I was thinking about that, I'll move it into Auton
Using paralleldeadlinegroup here is a little detrimental to the sequence's readability in my opinion, but I'll write it in